Source/Core/DiscIO/DiscUtils.h

```diff
@@ -49,6 +50,24 @@ constexpr u32 WII_NONPARTITION_DISCHEADER_SIZE = 0x100;
 constexpr u32 WII_REGION_DATA_ADDRESS = 0x4E000;
 constexpr u32 WII_REGION_DATA_SIZE = 0x20;
 
+// Default block size for GCZ/RVZ images
+constexpr int GCZRVZ_DEFAULT_BLOCK_SIZE = 0x20000; // 128 KiB
+
+// Because DVD timings are emulated as if we can't read less than an entire ECC block at once
+// (32 KiB - 0x8000), there is little reason to use a block size smaller than that.
+constexpr int GCZRVZ_MIN_BLOCK_SIZE = 0x8000; // 32 KiB
+
+// For performance reasons, blocks shouldn't be too large.
+// 2 MiB (0x200000) was picked because it is the smallest block size supported by WIA.
+constexpr int GCZRVZ_MAX_BLOCK_SIZE = 0x200000; // 2 MiB
+
+// If we didn't find a good block size, pick the block size which was hardcoded
+// in older versions of Dolphin. That way, at least we're not worse than older versions.
+constexpr int GCZ_FALLBACK_BLOCK_SIZE = 0x4000;
+
+// This is the smallest block size supported by WIA. For performance, larger sizes are avoided.
```

Comment:
The `For performance, larger sizes are avoided.` part of the comment isn't especially relevant in this context. Probably best to remove it. We already have a comment further up about how "For performance reasons, blocks shouldn't be too large".